tools/wheelmaker.py

```diff
@@ -398,6 +404,9 @@ def main() -> None:
                       encoding="utf-8") as metadata_file:
                 metadata = metadata_file.read()
 
+        if arguments.requires_file:
+            with open(arguments.requires_file) as fp:
```

Comment:
Shouldn't this instead be written to `Requires-Dist:` in the metadata? My use case for this is to be able to define 3rdparty dependencies in `requirements.in` files and be able to continue to use the existing `requires` field for local dependencies.